versions/draft-3/SPEC.md

```diff
@@ -509,6 +518,12 @@ Below are the valid results for operators on types.  Any combination not in the
 |`String`|`>=`|`String`|`Boolean`||
 |`String`|`<`|`String`|`Boolean`||
 |`String`|`<=`|`String`|`Boolean`||
+|`FileURI`|`+`|`FileURI`|`FileURI`|Append FileURI paths|
```

Comment:
I don't think addition makes sense. I know `File` has it (which also has never made sense to me), but unless this is something you can see how you'll want to use, I'd just remove it.

Reply:
Removed.

Done